packages/cf.js/considerations-for-0-0-2.md

```diff
@@ -0,0 +1,13 @@
+# missing features of v0.0.2
+
+- need a way to provide a `timeout` during the install flow
+    - argument to `proposeInstall`?
+    - attribute of the manifest.json
+- currently, the user's address is never being set
+    - provide as an argument: `new Client(nodeProvider, address)`
```

Comment:
I'm failing to see how a user's address is relevant to the initialization of a _Client_.

What's the intent of providing an address here?

Reply:
We were using it previously to provide the fromAddress in our action requests, but it sounds like the node will won't need that information any longer.